### ISSUE TITLE
It can't be displayed at the end of line in Android studio

### ISSUE THREAD

**Issue by AndrewTseZhou** [zielu/GitToolBox#303]
Android Studio: 4.1.3
OS version: Mac OS Big Sur 11.2.3

Problem:  When using Android Studio to write flutter project, dart file often appears the problem that git submission information cannot be displayed at the end of line

**Comment by zielu** [comment]
Hi, I'll take a look

**Comment by AndrewTseZhou** [comment]
> Hi, I'll take a look.
> 
> @AndrewTseZhou Please check if you have git roots correctly setup: https://github.com/zielu/GitToolBox/wiki/Manual#inline-blame-does-not-work-in-git-directory

Yes, I have git roots correctly. When appears this problem, I have to close the file, and reopen it, then inline blame work.

**Comment by zielu** [comment]
Checkout one of 202 plugin line - in should be available in android studio 4.2.1
